### DISCUSSION TITLE
another domain name(url) for shared docs

### DISCUSSION THREAD

**Discussion by dantesia** [outline/outline#6992] [Self Hosting] [answered]
There is a portal for example test.pro. If you share a document, it will have an address, for example, test.pro/s/wiki 

Is it possible to make sure that on wiki.test.pro (via a proxy) we show test.pro/s/wiki/doc1 without redirects? and the path turned out to be wiki.test.pro/doc1, etc.

**Comment by tommoor** [comment, marked as answer]
For self-hosted you could write this logic in your proxy to rewrite the requests to the backend and enable something like this.

For hosted at getoutline.com we've done this work, so it's possible if you email support to get this behavior.

**Comment by Sapkotaanish** [comment]
@tommoor Could you please pass the proxy configuration or there details to achieve this?
I am getting login page while trying to proxy pass to the published page. (In self-hosted)
Thanks.
